### PR TITLE
Review for #84

### PR DESCRIPTION
**Pull request by unode**
Minor corrections during issue #84

### CODE REVIEW

**Comment by tobyhodges** [comment]
I've marked this as a draft so it won't get merged before you're ready. Mark it "ready for review" once you're done 🙂

**Comment by unode** [comment]
Ah!, my overuse of GitLab creeping up in this PR title :smile:
Thanks Toby

File: _includes/links.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 [email]: mailto:team@carpentries.org
 [filters]: https://jekyllrb.com/docs/liquid/filters/
 [ghp-dependencies]: https://pages.github.com/versions/
+[git]: https://git-scm.com/
```
Comment (tobyhodges) [inline _includes/links.md:21]: I guess we should link to the git homepage, but please also find a way to add a link to https://swcarpentry.github.io/git-novice/ in this section

Comment (unode) [reply _includes/links.md:21]: Added.

File: _episodes/includes.md [hunk 2]
```diff
@@ -149,7 +149,7 @@ at the top of your page.
 > > ## Solution
 > >
 > > ```
-> > <a href="https://YOUR_USERNAME.github.io/YOUR_REPO_NAME/"><img src="/images/site_banner.png" alt="Group Website with Jekyll"></a>
+> > <a href="https://YOUR_USERNAME.github.io/YOUR_REPO_NAME/"><img src="images/site_banner.png" alt="Group Website with Jekyll"></a>
```
Comment (unode) [inline _episodes/includes.md]: I can see this break if the include done later is done from another folder. Either way, neither prefixing with / nor leaving it out is going to work for all cases.

Comment (unode) [reply _episodes/includes.md]: Wondering if it makes sense to turn this into an exercise in "Page layouts" as part of "Expanding the layout".

**Comment by unode** [comment]
Sorry for the delay in wrapping this up.

Noticed after that @annefou did the review in issues, some of the things in this pull request have been pointed out in #113, #114, #115 and #116 as well.

**Comment by unode** [comment]
Fixed conflicts. Ready for review/merge

File: _episodes/github-pages.md [hunk 3]
```diff
@@ -168,8 +169,8 @@ We are now ready to start adding more content to our website. Let's do some exer
 > >
 > >        Twitter: [@my_research_project](https://twitter.com/my_research_project)
 > >
-> >     Note how we used various Markdown syntax: quoted text ('>'), italic font ('*') and external links
-> >     (square '[]' and round brackets '()' for mailto and regular Web URLs).
+> >     Note how we used various Markdown syntax: quoted text (`>`), italic font (`*`) and external links
+> >     (square `[]` and round brackets `()` for mailto and regular Web URLs).
```
Comment (tobyhodges) [inline _episodes/github-pages.md]:
```suggestion
> >     (a combination of square `[]` and round brackets `()` containing the link text and mailto or regular Web URLs respectively).
```

I think this is clearer - the original wording could potentially be confusing as a learner might read it as meaning "use square brackets for mailto and regular brackets for Web URLs."

**Review by tobyhodges — APPROVED** [review]
One suggested change but otherwise this looks great. Thanks @unode

**Pull request merged**